src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length ? (
+      comments.map(comment => (
+        <CommentInfo comment={comment} key={comment.id} />
+      ))
+    ) : (
+      <strong data-cy="NoCommentsMessage">No comments yet</strong>
+    )}
```

Comment:
move this logic where you call `CommentList`

comments.length ? CommentList : 'no comments'

Reply:
comment list should render only list